app/models/goal.py

```diff
@@ -1,5 +1,31 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
 from ..db import db
+from flask import abort, make_response
 
 class Goal(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
+    tasks: Mapped[list["Task"]] = relationship(back_populates="goal")
+
+    def obj_to_dict(self):
+        goal_as_dict = {}
+        goal_as_dict["id"] = self.id
+        goal_as_dict["title"] = self.title
+
+        return goal_as_dict
```

Comment:
Can also directly return a dictionary literal
```suggestion
 return {
            "id": self.id,
            "title": self.title
        }
```